pandas/io/sql.py

```diff
@@ -1655,7 +1655,7 @@ def read_table(
         SQLDatabase.read_query
 
         """
-        self.meta.reflect(bind=self.con, only=[table_name])
+        self.meta.reflect(bind=self.con, only=[table_name], views=True)
```

Comment:
guessing this isnt too expensive performance wise that we will now start reflecting all tables by default (include views/mat views) just thinking if this will impact performance for current users.

Reply:
Yeah I'm assuming it's negligible relative to the query or IO, especially since the old implementation was able to access views too